src/assets/styles/layout.scss

```diff
@@ -11,17 +11,20 @@ body,
 
 .layout-main {
   flex: 1;
-  display: flex;
 }
 
 .event-map,
 .event-list {
   display: none;
-  flex: 1;
-  flex-direction: column;
-  &.-active {
-    display: flex;
-  }
+}
+
+.event-list.-active {
```

Comment:
Not sure what exactly happened, but I think something around here...at desktop widths I am not seeing the event list on the right hand side